okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/OptionalMethod.java

```diff
@@ -169,6 +169,12 @@ private Method getMethod(Class<?> clazz) {
   private static Method getPublicMethod(Class<?> clazz, String methodName, Class[] parameterTypes) {
     Method method = null;
     try {
+      if(clazz == null) {
```

Comment:
why would clazz be null here?

Also, java style says add a space between `if` and `(`